W-20624073 forward compatibility review (#299)
mulesoft/docs-mule-sdk: modules/ROOT/pages/choosing-version.adoc

endif::[]
:keywords: mule, sdk, create, new, project, getting, started, version

> Starting with mule-java-extension-parent version 1.11.0, the SDK supports forward compatibility. The SDK decouples the SDK version from the Minimum Mule Version your module requires, so you can always use the latest SDK version while controlling runtime compatibility independently.

Author: ```suggestion
Starting with `mule-java-extension-parent` version 1.11.0, the SDK supports forward compatibility. The SDK decouples the SDK version from the Minimum Mule Version your module requires, so you can always use the latest SDK version while controlling runtime compatibility independently.
```

> Starting with `mule-java-extension-parent` version 1.11.0, the SDK supports forward compatibility. The SDK decouples the SDK version from the Minimum Mule Version your module requires, so you can always use the latest SDK version while controlling runtime compatibility independently.

Reviewer: ```suggestion
Starting with `mule-java-extension-parent` version 1.11.0, the SDK supports forward compatibility. The SDK version is now decoupled from the Minimum Mule Version your module requires, so you can always use the latest SDK version while controlling runtime compatibility independently.
```
what do you think?

> == Recommended Approach

Author: ```suggestion
```
